.env.example

```
@@ -465,3 +465,20 @@ FASTER_WHISPER_BASE_URL=http://faster-whisper:8000
+# host, or download the `agent-engine-sif` CI artifact. See
+# deploy/apptainer/README.md. Leave blank on the WSL2 dev box (no apptainer).
+HIVE_AGENT_SIF_PATH=
+
+# ── Agent engine (Apptainer sandbox runtime) ─────────────────────────────────
+# HIVE_AGENT_SIF_PATH: Absolute path to the prebuilt agent-engine Apptainer
+#   image. agent-engine launches each session inside this image and refuses to
+#   start without it. The image is linux/amd64 only and cannot be built on the
+#   WSL2 dev box; build it on a demo/prod host with `make agent-sif`, or
+#   download the CI-built `agent-engine-sif` artifact. See
+#   deploy/apptainer/README.md. Leave blank on hosts that do not run agents.
+HIVE_AGENT_SIF_PATH=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The HIVE_AGENT_SIF_PATH key is duplicated

(DuplicatedKey)